functions.yml

```diff
@@ -187,26 +201,23 @@ etl:
           doc: Location of the chain file on each node in the cluster
           type: str
         - name: minMatchRatio
-          doc: Minimum fraction of bases that must remap to lift over successfully
+          doc: Minimum fraction of bases that must remap to do liftover successfully. If not provided, defaults to 0.95.
           type: double
           is_optional: true
+      returns: A struct containing ``contigName``, ``start``, and ``end`` fields after liftover.
     - name: normalize_variant
       doc: |
-             Normalize the variant with a behavior similar to vt normalize or bcftools norm.
+             Normalizes the variant with a behavior similar to vt normalize or bcftools norm.
              Creates a StructType column including the normalized start, end, referenceAllele and
```

Comment:
It may make more sense to move this extensive description to its own section in the variant normalization doc page

Reply:
Left it in as we talked. Will address var normalization doc in a followup PR